Sources/Kitura/Router.swift

```diff
@@ -625,7 +625,7 @@ extension Router : ServerDelegate {
     /// - Parameter response: The `RouterResponse` object used to respond to the
     ///                     HTTP request.
     private func sendDefaultResponse(request: RouterRequest, response: RouterResponse) {
-        if request.parsedURLPath.path == "/" {
+        if elements.isEmpty && request.parsedURLPath.path == "/" {
```

Comment:
I agree with the idea of disabling the built-in resource file server / welcome page, but I'm not sure that doing so strictly in the presence of routes will work for us.

Today, if someone runs `kitura init` then a sample project is created with a `/health` route; from `kitura create` there may also be some example routes.  I wouldn't want to suppress the default welcome page in this scenario.

Perhaps a conscious opt-in or opt-out mechanism might work, possibly in conjunction with this change.